pkg/core/native/native_neo.go

```diff
@@ -232,6 +232,10 @@ func newNEO(cfg config.ProtocolConfiguration) *NEO {
 	md = newMethodAndPrice(n.getCommittee, 1<<16, callflag.ReadStates)
 	n.AddMethod(md, desc)
 
+	desc = newDescriptor("getCommitteeAddress", smartcontract.Hash160Type)
```

Comment:
It's only available after the Cockatrice hardfork, so at the very minimum this needs to be updated in #3213.

Reply:
I saw the Cockatrice attribute, but since we don't have it now, I've merged this PR as is. I'll search for all mentions of `Cockatrice` in C# node anyway while implementing it.